python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -9,18 +9,29 @@
 from functools import reduce
 from typing import TYPE_CHECKING, Any, ClassVar
 
-import cudf_polars.experimental.groupby
-import cudf_polars.experimental.io
-import cudf_polars.experimental.join
-import cudf_polars.experimental.select
-import cudf_polars.experimental.shuffle  # noqa: F401
```

Comment:
I *think* removing these lines will break a lot of things, because we are registering dispatch functions in these modules.